android/app/build.gradle

```diff
@@ -0,0 +1,37 @@
+apply plugin: 'com.android.application'
+apply plugin: 'kotlin-android'
+apply plugin: 'kotlin-android-extensions'
+
+android {
+    compileSdkVersion 29
+    buildToolsVersion "29.0.2"
+
+    defaultConfig {
+        applicationId "com.sample"
+        minSdkVersion 21
+        targetSdkVersion 29
+        versionCode 1
+        versionName "1.0"
+        testInstrumentationRunner "androidx.test.runner.AndroidJUnitRunner"
+    }
+}
+
+def hermesPath = "node_modules/hermes-engine/android"
```

Comment:
Maybe we can reuse the node module resoultion algorithm used in RN: https://github.com/facebook/react-native/blob/a35efb94006bfa3f541bf3fc3ab5262740f00525/ReactAndroid/build.gradle#L223

Reply:
Awesome! Will change it in a bit.

Reply:
Done! Awesome suggestion.